extensions/rehooks/package.json

```diff
@@ -11,10 +11,17 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "search-hooks",
```

Comment:
Can we keep the old `name`? It is the unique ID of the command where ranking, aliases, and hotkeys are saved. You can, of course, change the title 🙂

Reply:
I don't have access to my laptop rn, but looks like you assigned this to yourself